README.md

```diff
@@ -507,6 +509,526 @@ source scripts/makeAllPlots.sh figures/ResponseUniformityMaps
 
 this will then execute genericPlotter taking each `*.cfg` file in the `figures/ResponseUniformityMaps` directory.
 
+### 3.b.ii.I Helper Script - Parsing Excel Files to genericPlotter 
+
+- **Main script:** Produce\_Config\_File.py
+- **Additional scripts:** QC3.py, QC4\_HV.py, QC4\_SS.py, QC5\_Gain.py, QC5\_Rate.py
+
+
+**Main script:** This tool is designed to read selected columns from an excel file and produce a config file in the genericPlotter input style described in session: [4.e.iv. Plot Config File](#4eiv-plot-config-file). Following, the produced config file can be used as an input file by genericPlotter to create a graph in the official CMS style.
```

Comment:
Typos and clarification changes needed:

> and produce a config file in the genericPlotter input style described in session:

To:

> and produce a config file for `genericPlotter` as described in section:

Reply:
Also white space reduction needed.

Reply:
done..

Reply:
done..